docs/getting_started/tutorial.md

```diff
@@ -238,8 +240,13 @@ class BlogIndexPage(Page):
 Run `python manage.py makemigrations` and `python manage.py migrate`.
 
 Since the model is called `BlogIndexPage`, the default template name
-(unless we override it) will be `blog/templates/blog/blog_index_page.html`. Create this file
-with the following content:
+(unless we override it) will be `blog_index_page.html` and the template path will be  
+`blog/templates/blog/blog_index_page.html`. In your new blog folder, do the following:
+1. Create a folder named `templates`.
+2. Create a folder named `blog` in your newly created `templates` folder.
+3. Create a file named `blog_index_page.html` in your newly created `blog` folder.
```

Comment:
Just thinking about this, we should try to assume some basic knowledge of how folder creation works and how a path to a file assumes you can create the folders if they are not present. Otherwise this tutorial may become too huge.

However, I agree that this could be worded better, here is maybe a refinement.

```suggestion
will be `blog_index_page.html` and Django will look for this template in your `blog` app folder within the `templates` directory and try to find a template with a name based on your Page model's name. The default behaviour can be overridden if needed.

Create a file at the location `blog/templates/blog/blog_index_page.html` and in that file enter the following content:
```

Note: line 249 can be removed below.